### ISSUE TITLE
[1.18.2] Updating JEI past 196 causes crash on worldLoading

### ISSUE THREAD

**Issue by drakray** [r8420/MoreOverlays-updated#27]
https://pastebin.com/9zJZWf3j
<details>
  <summary>Crash excerpt</summary>

```
Description: Unexpected error

java.lang.NoClassDefFoundError: mezz/jei/gui/overlay/IngredientListOverlay
	at at.ridgo8.moreoverlays.itemsearch.JeiModule.updateModule(JeiModule.java:30) ~[moreoverlays-1.20.5-1.18.2.jar%23208!/:1.20.5-mc1.18.1] {re:classloading}
	at at.ridgo8.moreoverlays.itemsearch.GuiHandler.onGuiInit(GuiHandler.java:30) ~[moreoverlays-1.20.5-1.18.2.jar%23208!/:1.20.5-mc1.18.1] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_2187_GuiHandler_onGuiInit_Post.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-5.0.3.jar%232!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-5.0.3.jar%232!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-5.0.3.jar%232!/:?] {}
  ```
  
</details>

Minecraft 1.18.2
Forge 1.18.2-40.1.51
More Overlays Updated 1.20.5-1.18.2
JEI 10.1.0.202, also with 10.1.0.200
buncha other mods

**Comment by ProsperCraft** [comment]
I didn't crash on world load on multiplayer server but my client did exit entirely with no crash on opening my inventory.

https://gist.githubusercontent.com/ProsperCraft/9e094139af80ab7efc127666be71794e/raw/ba586cdeda1c1ec09d8e2521844f048380fb974c/gistfile1.txt

**Comment by r8420** [comment]
Fixed in MoreOverlays 1.20.6.
Download [here](https://www.curseforge.com/minecraft/mc-mods/more-overlays-updated/files/all?filter-status=1&filter-game-version=2020709689%3A9008)
